_partials/_cloud-mst-comparison.md

```diff
@@ -3,11 +3,12 @@
   header="Timescale Cloud or MST?"
 >
 
-Timescale Cloud is a high-performance developer focused cloud that provides PostgreSQL services enhanced 
-with our blazing fast vector search. You can securely integrate Timescale Cloud with your AWS, GCS or Azure 
-infrastructure. [Create a Timescale Cloud service][timescale-service] and try for free.
+Both the Managed Service for TimescaleDB (MST) by Aiven and Timescale Cloud are TimescaleDB services built on 
```

Comment:
Timescale Cloud and Managed Service for TimescaleDB (MST)  instances are TimescaleDB extension hosted on PostgreSQL. TimescaleDB implements relational <SOMETHING>, time-series <SOMETHING>, vector search, and more.

Reply:
> Timescale Cloud and Managed Service for TimescaleDB (MST) instances are TimescaleDB extension hosted on PostgreSQL.
This positions Timescale Cloud as hosted TimescaleDB while we offer a lot more (PopSQL, query performance insights, etc.).

I think we can say that both offer Postgres databases that include the TimescaleDB extension or something like that. Let's be careful not to say or imply Timescale Cloud is just a managed hosted version of TimescaleDB because it's not.